composer.json

```diff
@@ -12,9 +12,9 @@
   "license": "MIT",
   "name": "pdsinterop/flysystem-rdf",
   "require": {
-    "php": "^7.3",
+    "php": "^8.0",
```

Comment:
I think we might want to use `"^7.3||^8.0"` or `">=7.1.0"`, as EasyRDF supports PHP>=7.1.

No need to drop PHP7 here yet, I think?

Reply:
Agreed, I think we should align it with nextcloud 24:
```
  "require": {
    "php": ">=7.3 <=8.1"
  }
```